src/main/webapp/components/lists/list-create.js

```diff
@@ -111,10 +119,55 @@ export const ListCreate = props => {
           { label: 'Yes', value: true },
           { label: 'No', value: false },
         ]}
-        value={list.useFilter === true}
-        onChange={e => setList({ ...list, useFilter: e })}
+        value={showFilters === true}
+        onChange={e => setShowFilters(e)}
       />
-      {/*TODO Add Filtering capablities*/}
+      {showFilters ? (
+        <React.Fragment>
+          <Box display="flex" alignItems="center" justifyContent="center">
+            <TextField
+              select
+              label="Type"
+              value={filterType}
+              onChange={e => {
+                setFilterType(e.target.value)
+                setList({
+                  ...list,
+                  list_cql: `{type=${filterType}, filters=${JSON.stringify(
+                    filters
+                  )}}`,
+                })
+              }}
+              style={{ height: '100%', width: '50%', padding: '10px' }}
+            >
+              <MenuItem key="AND" value="AND">
+                AND
+              </MenuItem>
+              <MenuItem key="OR" value="OR">
+                OR
+              </MenuItem>
+            </TextField>
+            <Button
+              color="primary"
+              variant="contained"
+              onClick={() => {
+                setFilters([...filters, anyTextFilter])
+                setList({
+                  ...list,
+                  list_cql: `{type=${filterType}, filters=${JSON.stringify(
+                    filters
+                  )}}`,
```

Comment:
since you're adding a filter here, should this be:
```suggestion
                setFilters([...filters, anyTextFilter])
                setList({
                  ...list,
                  list_cql: `{type=${filterType}, filters=${JSON.stringify(
                    [...filters, anyTextFilter]
                  )}}`,
```